packages/@aws-cdk/aws-glue/test/integ.job.ts

```diff
@@ -2,48 +2,72 @@ import * as path from 'path';
 import * as cdk from '@aws-cdk/core';
 import * as glue from '../lib';
 
+/**
+ * To verify the ability to run those jobs
+ *
+ * Run the job using
+ *   `aws glue start-job-run --job-name <job name>`
```

Comment:
I still have more to do here

Reply:
Found 2 bugs when trying to run jobs created by the integ tests
1. Glue Role does not have access to the assets bucket
2. Python shell job is failing saying Glue Version 2.0 is unsupported. Trying to add a Python shell via console shows Glue 1.0 only
![Screenshot 2021-09-08 at 17 52 24](https://user-images.githubusercontent.com/5001/132554584-beeb3cd6-0cc4-4612-956a-5551c0a00faa.png)

As such
1. I'll have a fix for the permissions
2. I'll only check that python shell is not supported with Glue 0.9 so as to not block any future work Glue might do to have the shell supported with Glue 2.0 or 3.0

Reply:
With the fixes in https://github.com/aws/aws-cdk/pull/12506/commits/cd2d2eebd2ed373f48074d5972001c866468dce5, all integ jobs can be started and they succeed